.github/workflows/release-container-image.yml

```diff
@@ -0,0 +1,237 @@
+name: Release multi-arch container image
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - "test/**"
+  schedule:
+    - cron: "0 0 * * *" # Run every day at 00:00 UTC.
+
+env:
+  REGISTRY: ghcr.io/${{ github.repository_owner }}
+  REGISTRY_USER: ${{ github.actor }}
+  REGISTRY_PASSWORD: ${{ github.token }}
+  IMAGE_NAME: dangerzone/dangerzone
+  BUILDKIT_IMAGE: "docker.io/moby/buildkit:v19.0@sha256:14aa1b4dd92ea0a4cd03a54d0c6079046ea98cd0c0ae6176bdd7036ba370cbbe"
```

Comment:
Should we actually get this from the same place we define the version of the tools we use? Currently this is [`Dockerfile.env`](https://github.com/freedomofpress/dangerzone/blob/main/Dockerfile.env), but maybe this can change so we are defining everything in the same location.

Reply:
Hm, currently this is copied verbatim from the `repro-build` script, so the source of truth is there. But we could do as you say, maybe with some comments on what's the purpose of `Dockerfile.env`.

Reply:
Yeah, and I don't like having duplicated strings like that, because they might get out of sync, and it's double work when updating it.

If the `Dockerfile.env` doesn't seem the right place, we could also could also rename it to something else.

Reply:
Done in af8ba7429425ea0027ae5adfb393e071d59a2f55.